src/App.jsx

```diff
@@ -14,33 +20,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function sortItems(goods, { sortType, queue }) {
+  const items = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  if (sortType) {
+    items.sort((a, b) => {
+      switch (sortType) {
+        case SORT_BY_ALPHABET:
+          return a.localeCompare(b);
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+        case SORT_BY_LENGTH:
+          return a.length - b.length;
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (queue) {
+    items.reverse();
+  }
+
+  return items;
+}
+
+export const App = () => {
+  const [queueField, setQueueField] = useState(false); // Булеве значення
+  const [sortField, setSortField] = useState('');
```

Comment:
The initial state of `sortField` is set to an empty string. Ensure that the `sortItems` function can handle this case properly, or consider setting a default sort type to prevent unexpected behavior.